### DISCUSSION TITLE
读书：哲学家们都干了些什么 — 码志

### DISCUSSION THREAD

**Discussion by giscus** [mzlogin/blog-comments#265] [Announcements]
# 读书：哲学家们都干了些什么 — 码志

https://mazhuang.org/2023/03/08/what-did-the-philosophers-do/

**Comment by yejg** [comment]
人生没有意义，但是追寻意义的过程有意义。 --- 不记得是谁说了的

**Reply by mzlogin** [reply]
> 用西西弗的比喻来说，我们只能在推石头的时候哄自己说这么做是有意义的，并且乐在其中。这个哄骗自己的借口，就是人生意义。

大概就是这个意思 :laughing:
